packages/client/package.json

```diff
@@ -48,9 +48,9 @@
     "@tanstack/react-query": "5.81.2",
     "@tanstack/react-query-devtools": "5.81.2",
     "@tanstack/react-query-persist-client": "5.81.2",
-    "@trpc/client": "11.4.2",
-    "@trpc/server": "11.4.2",
-    "@trpc/tanstack-react-query": "11.4.2",
+    "@trpc/client": "11.4.3",
```

Comment:
Updated these just in case zod's branded types would have worked a little better